pkgs/top-level/python-packages.nix

```diff
@@ -9148,6 +9148,24 @@ in {
     };
   };
 
+  pysrt = buildPythonApplication rec {
```

Comment:
Sorry, I read somewhere else (nixpkgs manual IIRC) that they should be added here, I'll move it.
`pysrt` is available both as a library and as an application, that's why it uses `buildPythonApplication`.